src/http/index.ts

```diff
@@ -0,0 +1,61 @@
+import { go } from '@api3/promise-utils';
+import axios, { type Method, type AxiosError, type AxiosResponse } from 'axios';
+import { pick } from 'lodash';
```

Comment:
Nit: I guess we can't always guarantee where this will be used and we might not always be fine with importing the entirety of lodash

```suggestion
import pick from 'lodash/pick';
```

Reply:
I've setup ESLint to prefer these style of imports (in previous PR).